src/test/kotlin/no/nav/syfo/api/endpoints/SenOppfolgingEndpointsSpek.kt

```diff
@@ -218,6 +233,75 @@ object SenOppfolgingEndpointsSpek : Spek({
                     testInvalidPersonIdent(ferdigbehandlingUrl, validToken, HttpMethod.Post)
                 }
             }
+
+            describe("POST /get-kandidater") {
```

Comment:
Burde vi hatt en test på at det er nyeste kandidat-raden for en person som hentes ut (hvis personen har blitt kandidat flere ganger) ?

Reply:
Yes har fikset det nå 👍🏼